ci/deploy.sh

```diff
@@ -22,14 +22,19 @@ git checkout "$TRAVIS_BRANCH"
 git remote set-url origin "git@github.com:$TRAVIS_REPO_SLUG.git"
 
 # Decrypt and add SSH key
+eval "$(ssh-agent -s)"
+if [ -v MANUBOT_SSH_PRIVATE_KEY ]; then
+  set +o xtrace  # TODO: better way of temporarily disabling xtrace
+  base64 --decode <<< "$MANUBOT_SSH_PRIVATE_KEY" | ssh-add -
```

Comment:
`-` at the end should be removed.

Reply:
Testing whether `-` can be removed in https://github.com/manubot/manubot.org/commit/ecf2af99ceff45833b67868cb1a033ab558c3385 / https://travis-ci.com/manubot/manubot.org/builds/117875833

Reply:
I tested it in the morning on my local desktop.